src/constants.js

```diff
@@ -152,6 +154,7 @@ export const PageTypes = {
 export const DefaultValues = {
   [SettingIds.ANON_CHAT]: false,
   [SettingIds.AUTO_THEATRE_MODE]: false,
+  [SettingIds.AUTO_LEAVE_RAIDS]: false,
```

Comment:
It may be more accessible to have this default on and only when this user untoggles this setting they will be opted out of raids. e.g. `[SettingIds.AUTO_JOIN_RAIDS]: true`